tests/transformers/test_semantic_similarity.py

```diff
@@ -64,7 +64,7 @@ def test_semantic_meta_fit():
 
     y = [1, 1, 0]
 
-    classifier.fit(X, y, epochs=3)
+    classifier.fit(X, y, epochs=5)
```

Comment:
how come you did this out of interest?

Reply:
the test was failing with loss increasing for some reason